Garrett-Iannuzzi/hot_like_sauce
# About section setup and style

## What does this PR do?
- Adds icons for the bottle
- adds text for about 
- positions the container

## How to test?
- pull down `about-section-setup`
- run `npm start` and see changes

## Issues:
- this PR closes #12 #9 

## Notes:
- None
